### PR TITLE
Design changes

### PR DESCRIPTION
**Pull request by slattery-mark**
Implements consistent styles across Index, Login, and Bookmarks as well as minor UI tweaks, changed formStyles.css to frostedStyles.css and refactored code to be mobile up

Progress toward [NR#10](https://github.com/WSU-4110/Eventsly/issues/53)

### CODE REVIEW

File: static/styles/createEventStyleSheet.css [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    background-color: #7b82b3;
+    cursor: pointer;
+}
+
+.create-event input[type='submit'].disable-submit {
+    opacity: 0.5;
+}
+
+/*
+* labels/counters/requirement text
```
Comment (CarbonatedSoftDrink) [inline static/styles/createEventStyleSheet.css:118]: the labels inside of the input boxes don't disappear or move when typing.

**Review by CarbonatedSoftDrink — APPROVED** [review]
besides my comment on the createEvent page, im ready to approve.

File: static/styles/createEventStyleSheet.css (hunk 1 above)
Comment (slattery-mark) [reply static/styles/createEventStyleSheet.css:118]: Yeah, the labels moving is part of the frontend validation scripts - I actually do have it fixed in the frontend_validation branch, I tried to keep this pull req strictly design stuff. The labels will move in my next pull req from frontend_validation

**Review by cj-fox — APPROVED** [review]
Good use of software design patterns.

**Pull request merged**